internal/signerverifier/dsse/dsse_test.go

```diff
@@ -28,49 +25,58 @@ func TestCreateEnvelope(t *testing.T) {
 }
 
 func TestSignEnvelope(t *testing.T) {
-	env, err := createSignedEnvelope()
+	keyPath := setupTestECDSAPair(t)
+
+	signer, err := loadSSHSigner(keyPath)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	assert.Len(t, env.Signatures, 1)
-	assert.Equal(t, "a0xAMWnJ3Hzf8j2zLFmniyUxV58m2lUprgzDPkJIRUORR4aKlX23WB3teaVMjXLuRKrD5GAMN8NSCR1vaetxBA==", env.Signatures[0].Sig)
-
-	// Try signing with the same key to ensure a new signature isn't appended
-	signer, err := signerverifier.NewSignerVerifierFromSecureSystemsLibFormat(signingKeyBytes) //nolint:staticcheck
+	env, err := createSignedEnvelope(signer)
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	assert.Len(t, env.Signatures, 1)
+	assert.Equal(t, "SHA256:oNYBImx035m3rl1Sn/+j5DPrlS9+zXn7k3mjNrC5eto", env.Signatures[0].KeyID)
+
 	env, err = SignEnvelope(context.Background(), env, signer)
 	assert.Nil(t, err)
 	assert.Len(t, env.Signatures, 1)
-	assert.Equal(t, "a0xAMWnJ3Hzf8j2zLFmniyUxV58m2lUprgzDPkJIRUORR4aKlX23WB3teaVMjXLuRKrD5GAMN8NSCR1vaetxBA==", env.Signatures[0].Sig)
+	assert.Equal(t, "SHA256:oNYBImx035m3rl1Sn/+j5DPrlS9+zXn7k3mjNrC5eto", env.Signatures[0].KeyID)
```

Comment:
with an RSA key, we could also test the value of the signature, right? Any reason for using ecdsa here? I imagine ed25519 was because of compat reasons.

Reply:
Yes, but it's a silly and no longer relevant reason related to https://github.com/gittuf/gittuf/pull/414/commits/a7aca9781cbc674a4c200287a802330d4847f2da:

> Depending on the version [and key type], ssh-keygen might require both public and
private key file, for (passwordless) export (-e) and sign (-Y sign)
operations.

TLDR: I switched to ecdsa, when rsa stopped working, after changing the test key file setup to only copy the  private key file. With the ecdsa private key file only it passed on linux/macOS but not on Windows, so ended up copying the key pair anyway.

If you want I can switch back to RSA. But I'd rather do that in a follow-up PR for improved testing for the new ssh key package.